web/static/js/components/thesis-editor.js

```diff
@@ -1,48 +1,41 @@
 import React from 'react'
```

Comment:
I moved this file to `./components`, since `thesis.js` is loading it as a component now.